mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_get_record_processing_log_function.sql

```diff
@@ -7,161 +7,168 @@ AS $$
 BEGIN
```

Comment:
Its hard to see the difference with code formatting applied in this sql file, maybe revert back formatting?

Reply:
Thank  you @SerhiiNosko for the mention! To hide whitespaces you could use "Hide whitespaces" feature
![Screen Shot 2022-01-28 at 11 59 42 AM](https://user-images.githubusercontent.com/88028273/151517726-cb473331-f3cd-457f-82cb-953587e6740a.png)
: